keyboards/1upkeyboards/1up_pi60_rgb_v2/info.json

```
@@ -0,0 +1,250 @@
+            {"matrix": [2, 6], "x": 108, "y": 32, "flags": 4},
+            {"matrix": [2, 7], "x": 123, "y": 32, "flags": 4},
+            {"matrix": [2, 8], "x": 138, "y": 32, "flags": 4},
+            {"matrix": [2, 9], "x": 153, "y": 32, "flags": 4},
+            {"matrix": [2, 10], "x": 168, "y": 32, "flags": 4},
+            {"matrix": [2, 11], "x": 183, "y": 32, "flags": 4},
+            {"matrix": [2, 13], "x": 207, "y": 32, "flags": 1},
+            {"x": 220, "y": 32, "flags": 2},
+            {"x": 220, "y": 45, "flags": 2},
+            {"matrix": [3, 13], "x": 203, "y": 45, "flags": 1},
+            {"matrix": [3, 10], "x": 175, "y": 45, "flags": 4},
+            {"matrix": [3, 9], "x": 161, "y": 45, "flags": 4},
+            {"matrix": [3, 8], "x": 146, "y": 45, "flags": 4},
+            {"matrix": [3, 7], "x": 135, "y": 45, "flags": 4},
+            {"matrix": [3, 6], "x": 116, "y": 45, "flags": 4},
+            {"matrix": [3, 5], "x": 101, "y": 45, "flags": 4},
+            {"matrix": [3, 4], "x": 86, "y": 45, "flags": 4},
+            {"matrix": [3, 3], "x": 71, "y": 45, "flags": 4},
+            {"matrix": [3, 2], "x": 56, "y": 45, "flags": 4},
+            {"matrix": [3, 1], "x": 41, "y": 45, "flags": 4},
+            {"matrix": [3, 0], "x": 17, "y": 45, "flags": 1},
+            {"x": 4, "y": 45, "flags": 2},
+            {"x": 4, "y": 57, "flags": 2},
+            {"matrix": [4, 0], "x": 9, "y": 57, "flags": 1},
+            {"x": 15, "y": 57, "flags": 2},
+            {"x": 23, "y": 57, "flags": 2},
+            {"matrix": [4, 1], "x": 28, "y": 57, "flags": 1},
+            {"x": 34, "y": 57, "flags": 2},
+            {"x": 41, "y": 57, "flags": 2},
+            {"matrix": [4, 2], "x": 47, "y": 57, "flags": 1},
+            {"x": 53, "y": 57, "flags": 2},
+            {"x": 67, "y": 57, "flags": 2},
+            {"x": 71, "y": 57, "flags": 1},
+            {"x": 81, "y": 57, "flags": 2},
+            {"x": 95, "y": 57, "flags": 2},
+            {"matrix": [4, 5], "x": 103, "y": 57, "flags": 1},
+            {"x": 110, "y": 57, "flags": 2},
+            {"x": 125, "y": 57, "flags": 2},
+            {"x": 134, "y": 57, "flags": 1},
+            {"x": 139, "y": 57, "flags": 2},
+            {"x": 153, "y": 57, "flags": 2},
+            {"matrix": [4, 9], "x": 159, "y": 57, "flags": 1},
+            {"x": 165, "y": 57, "flags": 2},
+            {"x": 172, "y": 57, "flags": 2},
+            {"matrix": [4, 10], "x": 177, "y": 57, "flags": 1},
+            {"x": 183, "y": 57, "flags": 2},
+            {"x": 190, "y": 57, "flags": 2},
+            {"matrix": [4, 12], "x": 196, "y": 57, "flags": 1},
+            {"x": 202, "y": 57, "flags": 2},
+            {"x": 209, "y": 57, "flags": 2},
+            {"matrix": [4, 13], "x": 215, "y": 57, "flags": 1},
+            {"x": 220, "y": 57, "flags": 2}
+        ]
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP28"
+    },
```

CI BuiId failures that are attributed to community layout/userspace keymaps would not be dismissed. What's imperative is the keyboard, configuration, and keymap(s) specific to this PR are functional/working - which they are 😉
```suggestion
    },
    "community_layouts" : ["60_ansi"],
```

---

When I put this line back in, then the build fails with this error:

🔴 1upkeyboards/1up_pi60_rgb_v2:brandonschlack-ansi

---

`brandonschlack-ansi` is a community layout keymap; any issues with it, thankfully, are not your responsibility to fix. Please configure/define this keyboard as it's intended 🙂